src/Raven.Server/RavenServer.cs

```diff
@@ -1507,10 +1507,10 @@ public bool CanAccess(string database, bool requireAdmin, bool requireWrite)
                 // worried about race conditions here if we move to HTTP 2.0 at some point. At that point,
                 // we'll probably want to handle this concurrently, and the cost of adding it in this manner
                 // is pretty small for most cases anyway
-                _caseSensitiveAuthorizedDatabases = new Dictionary<string, DatabaseAccess>(_caseSensitiveAuthorizedDatabases)
-                {
-                    {database, mode}
-                };
+                var authorizedDatabases = new Dictionary<string, DatabaseAccess>(_caseSensitiveAuthorizedDatabases);
+                    authorizedDatabases.TryAdd(database, mode);
```

Comment:
formatting + comment above is now irrelevant and can be removed?

Reply:
Done